apps/web/src/app/[locale]/[username]/page.tsx

```diff
@@ -11,8 +11,7 @@ export default function Page({ params }: Props) {
 }
 
 export async function generateMetadata({ params: { username } }: Props) {
-  // remove the %40 from the username
-  const name = username.substring(3);
+  const name = decodeURIComponent(username).substring(1);
```

Comment:
why decode just to remove?

Reply:
some people hve spaces in their name (from seed data, @Alexandra Osinski) and the spaces are %20 which is ugly

Reply:
i dont believe github allows spaces in usernames so this should never happen

Reply:
yeah that's true, I just noticed the %20 in prod at /@Alexandra Osinski (there may be others as well) from the seed data and it bugged me

it won't happen with just github accounts, but it will with the seed data

should I remove it?

Reply:
you are good to decode here.